extensions/interactions/FractionInput/directives/FractionInputValidationService.js

```diff
@@ -187,15 +192,20 @@ oppia.factory('FractionInputValidationService', [
             }
             for (var k = 0; k < ranges.length; k++) {
               if (isEnclosedBy(range, ranges[k])) {
-                warningsList.push({
-                  type: WARNING_TYPES.ERROR,
-                  message: (
-                    'Rule ' + (j + 1) + ' from answer group ' +
-                    (i + 1) + ' will never be matched because it ' +
-                    'is made redundant by rule ' + ranges[k].ruleIndex +
-                    ' from answer group ' + ranges[k].answerGroupIndex +
-                    '.')
-                });
+                var rule2 = answerGroups[ranges[k].answerGroupIndex - 1]
+                  .rules[ranges[k].ruleIndex - 1];
+                if(rule.type !== 'IsExactlyEqualTo' ||
```

Comment:
For future reference: add space before "("

Reply:
Oops missed these 2 styling fixes. Sorry about that. Shall I fix it separately right now?